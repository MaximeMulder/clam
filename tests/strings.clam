def main = {
    print("Hello");
    print("Hello" ++ " " ++ "world" ++ " !");

    print("Hello" == "Hello");
    print("Hello" == "Hello world !");
};
