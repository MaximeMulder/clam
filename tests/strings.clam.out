"Hello"
"Hello world !"
true
false
